src/library/Uno.Material/Styles/Application/Colors.xaml

```diff
@@ -186,6 +193,31 @@
 	<SolidColorBrush x:Key="SecondaryColor02Brush"
 					 Color="{StaticResource SecondaryColor02}" />
 
+	
+	<SolidColorBrush x:Key="SelectControlBackgroundColor"
```

Comment:
We should only keep the color palette here, control specific colors should go in their own resource dictionary. In this case they might be shared between 2-3 controls so I suggest a resourceDictionary SelectionControlColors.xaml and merge it where you need it